### PR TITLE
feat: add more kill event data

### PR DESCRIPTION
**Pull request by akiver**
Hi,

It adds more data to the kill event.

Thank you

### CODE REVIEW

File: pkg/demoinfocs/game_events.go [hunk 1]
```diff
@@ -336,6 +336,11 @@ func (geh gameEventHandler) playerDeath(data map[string]*msg.CSVCMsg_GameEventKe
 		IsHeadshot:        data["headshot"].GetValBool(),
 		PenetratedObjects: int(data["penetrated"].GetValShort()),
 		Weapon:            geh.getEquipmentInstance(killer, wepType),
+		AssistedFlash:     data["assistedflash"].GetValBool(),
+		AttackerBlind:     data["attackerblind"].GetValBool(),
+		NoScope:           data["noscope"].GetValBool(),
+		ThruSmoke:         data["thrusmoke"].GetValBool(),
```
Comment (markus-wa) [inline pkg/demoinfocs/game_events.go:342]:
```suggestion
		ThroughSmoke:         data["thrusmoke"].GetValBool(),
```

as above

**Comment by markus-wa** [comment]
Thanks @akiver :slightly_smiling_face: - I've added two suggestions, let me know what you thnk

File: pkg/demoinfocs/events/events.go [hunk 2]
```diff
@@ -140,6 +140,11 @@ type Kill struct {
 	Assister          *common.Player
 	PenetratedObjects int
 	IsHeadshot        bool
+	AssistedFlash     bool
+	AttackerBlind     bool
+	NoScope           bool
+	ThruSmoke         bool
```
[the remark this answers is not in the document]
Comment (akiver) [reply pkg/demoinfocs/events/events.go]: 👍 done

**Comment by markus-wa** [comment]
:tada:

**Pull request merged**